extensions/wikia/WikiFactory/WikiFactory.php

```diff
@@ -1223,21 +1244,21 @@ static public function getLocalEnvURL( $url ) {
 		}
 
 		$server = str_replace( '.' . $devbox . '.wikia-dev.com', '', $server );
-		$server = str_replace( '.wikia.com', '', $server );
+		$server = str_replace( self::WIKIA_COM, '', $server );
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Use "static" keyword instead of "self". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS2037)